yb-voyager/cmd/importData.go

```diff
@@ -206,6 +207,43 @@ func getYBServers() []*tgtdb.Target {
 	return targets
 }
 
+func fetchParllelJobs(targets []*tgtdb.Target) int {
+	totalCores := 0
+	targetCores := 0
+	queriedTargets := 0
+	for _, target := range targets {
+		conn, err := pgx.Connect(context.Background(), target.Uri)
```

Comment:
With LB, all `target.Uri` are going to point to the same IP. In that case, we don't know to which instance the request will hit.

Reply:
Is there a scenario where, on YBM, there is a difference in the number of cores for any pair of nodes?

Reply:
There can be e.g. user deployed a cluster with N cores per node. After few days, the user added a new node to the cluster with M cores.
But we don't need to be precise in this particular case. Aprox core count is also good enough for us.